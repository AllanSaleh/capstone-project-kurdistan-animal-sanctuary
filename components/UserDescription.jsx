import React from "react";
import { useTranslation } from "next-i18next";
import PropTypes from "prop-types";

const UserDescription = ({ about }) => {
  const { t } = useTranslation("userProfile");
  return (
    <div className="  ">
      <h1 className=" font-bold text-primary text-3xl pb-8">
        {t("userProfile.aboutMe")}
      </h1>
      <p className="text-sm md:text-lg text-primary">{about}</p>
    </div>
  );
};
UserDescription.propTypes = {
  about: PropTypes.string.isRequired,
};
export default UserDescription;
