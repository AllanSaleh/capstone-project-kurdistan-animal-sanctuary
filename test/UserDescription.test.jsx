import React from "react";
import renderer from "react-test-renderer";
import { test, expect } from "@jest/globals";
import UserDescription from "../components/UserDescription";

test("proparly renders the UserDescription component", () => {
  const tree = renderer.create(<UserDescription />).toJSON();
  expect(tree).toMatchSnapshot();
});
