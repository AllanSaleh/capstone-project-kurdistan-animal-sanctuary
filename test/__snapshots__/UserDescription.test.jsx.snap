// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`proparly renders the UserDescription component 1`] = `
<div
  className="  "
>
  <h1
    className=" font-bold text-primary text-3xl pb-8"
  >
    userProfile.aboutMe
  </h1>
  <p
    className="text-sm md:text-lg text-primary"
  />
</div>
`;
